main.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+from multiprocessing import Process
+from typing import TYPE_CHECKING
+
+from models.configuration import ServicesConfiguration
+from models.logger import create_logger
+from models.models_file import AppConfiguration
+from server import Server
+
+if TYPE_CHECKING:
+    from typing import List
+
+if len(sys.argv) == 1: sys.argv.append(os.getcwd())
+
+if "logs" not in os.listdir(sys.argv[1]): os.mkdir(os.path.join(sys.argv[1], "logs"))
+
+
+def main(config) -> 'List[Server]':
+
+    configuration = ServicesConfiguration(config.mockoon_file)
+    environments = configuration.load_configuration()
+    servers = [Server.factory(config.host, environment, debug=config.flask_debug, logging_level=config.logging_level)
+               for environment in environments]
+
+    return servers
```

Comment:
to mi się trochę nie podoba, po `main` oczekiwał bym, że odpalam metodę i już reszta sama się dzieje, tego `main`a przemianował bym na coś w rodzaju serverBuilder, a to co jest na końcu pliku w IFie wrzucił bym do `main`